### ISSUE TITLE
[porting] System.Drawing.Common uses its own InvalidEnumArgumentException, breaking tests

### ISSUE THREAD

**Issue by hughbe** [dotnet/corefx#20897]
The following test passes with netfx:

```cs
[Theory]
[InlineData(StringTrimming.None - 1)]
[InlineData(StringTrimming.EllipsisPath + 1)]
public void Trimming_SetInvalid_ThrowsInvalidEnumArgumentException(StringTrimming trimming)
{
    var format = new StringFormat();
    Assert.Throws<InvalidEnumArgumentException>("value", () => format.Trimming = trimming);
}
```

**Comment by hughbe** [comment]
I have a fix, it requires adding a reference to `System.ComponentModel.TypeConverter`

**Comment by mellinoe** [comment]
This was on my list of TODO's. I think we should move `InvalidEnumArgumentException` into a lower-level contract. There shouldn't be too much of a problem with that, I think.

**Comment by hughbe** [comment]
I've tried it in #20898 - could it be as simple?

**Comment by mellinoe** [comment]
We don't want System.Drawing.Common to reference TypeConverter itself -- eventually we may want the opposite dependency, actually. The TypeConverters in System.Drawing have all been removed for now, but if we add them back they will go into System.ComponentModel.TypeConverter. We did the same for System.Drawing.Primitives.

Moving `InvalidEnumArgumentException` into something that both libraries can reference should work, though.

**Comment by hughbe** [comment]
Understood. What about System.ComponentModel.Primitives?

**Comment by hughbe** [comment]
Actually got fixed in #20914
